### PR TITLE
add OrderedBucketListTest

### PR DESCRIPTION
**Pull request by ShatalovYaroslav**

### CODE REVIEW

File: src/test/java/org/ow2/proactive/OrderedBucketListTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    @Test
+    public void testOrderedBucketList() throws IOException {
+
+        List<String> fileLinesList = Files.readAllLines(Paths.get(ORDERED_BUCKET_LIST_FILE_NAME));
+
+        assertThat("The ordered bucket list file should present in project: " + ORDERED_BUCKET_LIST_FILE_NAME,
+                   fileLinesList,
+                   notNullValue());
+
```
Comment (Mykhailenko) [inline src/test/java/org/ow2/proactive/OrderedBucketListTest.java]: Are you sure Files.readAllLines will return null if there is nothing?

File: src/test/java/org/ow2/proactive/OrderedBucketListTest.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        assertThat("The ordered bucket list file should contain only 1 line: ", fileLinesList.size(), is(1));
+
+        assertThat("The ordered bucket list should not contain spaces: ",
+                   fileLinesList.get(0).contains(" "),
+                   is(false));
```
Comment (Mykhailenko) [inline src/test/java/org/ow2/proactive/OrderedBucketListTest.java]: some clarification here would be helpfull. why do you take only first line. Maybe fileLinesList is empty..

File: src/test/java/org/ow2/proactive/OrderedBucketListTest.java (hunk 1 above)
Comment (ShatalovYaroslav) [reply src/test/java/org/ow2/proactive/OrderedBucketListTest.java]: you are right, the method will throw the exception. So no need for this check

File: src/test/java/org/ow2/proactive/OrderedBucketListTest.java (hunk 2 above)
Comment (ShatalovYaroslav) [reply src/test/java/org/ow2/proactive/OrderedBucketListTest.java]: It should be only 1 line. Exactly this it's checked in previous assertion

Comment (Mykhailenko) [reply src/test/java/org/ow2/proactive/OrderedBucketListTest.java]: ok right

**Pull request merged**